src/lib/coingecko/utils.ts

```diff
@@ -14,7 +14,6 @@ import {
   thorchainChainId,
   toAssetId,
 } from '@shapeshiftoss/caip'
-import type { CoingeckoAssetPlatform } from '@shapeshiftoss/caip/src/adapters'
```

Comment:
import from `caip/src/adapters` is v. bad and broken, seemingly not so bad here because types only, but might as well for consistency